framework/mesh/mesh.cc

```diff
@@ -15,297 +15,4 @@ GetCurrentMesh()
-{
-  OpenSnLogicalErrorIf(vertices.empty(), "Empty vertex list.");
-
-  // Reorient 1D verts along z
-  std::vector<Vertex> zverts;
-  zverts.reserve(vertices.size());
-  for (double z_coord : vertices)
-    zverts.emplace_back(0.0, 0.0, z_coord);
-
-  // Create unpartitioned mesh
-  auto umesh = std::make_shared<UnpartitionedMesh>();
-  umesh->SetDimension(1);
-
-  // Create vertices
-  size_t Nz = vertices.size();
-
-  umesh->SetOrthoAttributes(1, 1, Nz - 1);
-  umesh->AddBoundary(4, "ZMAX");
-  umesh->AddBoundary(5, "ZMIN");
-
-  umesh->Vertices().reserve(Nz);
-  for (auto& vertex : zverts)
-    umesh->Vertices().push_back(vertex);
-
-  // Create cells
-  for (size_t c = 0; c < (zverts.size() - 1); ++c)
-  {
-    auto cell = new UnpartitionedMesh::LightWeightCell(CellType::SLAB, CellType::SLAB);
-
-    cell->vertex_ids = {c, c + 1};
-
-    UnpartitionedMesh::LightWeightFace left_face;
-    UnpartitionedMesh::LightWeightFace rite_face;
-
-    left_face.vertex_ids = {c};
-    rite_face.vertex_ids = {c + 1};
-
-    if (c == 0)
-      left_face.neighbor = 5 /*ZMIN*/;
-    if (c == (zverts.size() - 2))
-      rite_face.neighbor = 4 /*ZMAX*/;
-
-    cell->faces.push_back(left_face);
-    cell->faces.push_back(rite_face);
-
-    umesh->AddCell(cell);
-  }
-
-  umesh->ComputeCentroids();
-  umesh->CheckQuality();
-  umesh->BuildMeshConnectivity();
-
-  unpartitionedmesh_stack.push_back(umesh);
-
-  return unpartitionedmesh_stack.size() - 1;
-}
-
-size_t
-CreateUnpartitioned2DOrthoMesh(std::vector<double>& vertices_1d_x,
-                               std::vector<double>& vertices_1d_y)
-{
-  OpenSnLogicalErrorIf(vertices_1d_x.empty() or vertices_1d_y.empty(), "Empty vertex list.");
-
-  // Create unpartitioned mesh
-  auto umesh = std::make_shared<UnpartitionedMesh>();
-
-  umesh->SetDimension(2);
-
-  // Create vertices
-  size_t Nx = vertices_1d_x.size();
-  size_t Ny = vertices_1d_y.size();
-
-  umesh->SetOrthoAttributes(Nx - 1, Ny - 1, 1);
-  umesh->AddBoundary(0, "XMAX");
-  umesh->AddBoundary(1, "XMIN");
-  umesh->AddBoundary(2, "YMAX");
-  umesh->AddBoundary(3, "YMIN");
-
-  typedef std::vector<uint64_t> VecIDs;
-  std::vector<VecIDs> vertex_ij_to_i_map(Ny, VecIDs(Nx));
-  umesh->Vertices().reserve(Nx * Ny);
-  uint64_t k = 0;
-  for (size_t i = 0; i < Ny; ++i)
-  {
-    for (size_t j = 0; j < Nx; ++j)
-    {
-      vertex_ij_to_i_map[i][j] = k++;
-      umesh->Vertices().emplace_back(vertices_1d_x[j], vertices_1d_y[i], 0.0);
-    } // for j
-  }   // for i
-
-  // Create cells
-  auto& vmap = vertex_ij_to_i_map;
-  for (size_t i = 0; i < (Ny - 1); ++i)
-  {
-    for (size_t j = 0; j < (Nx - 1); ++j)
-    {
-      auto cell =
-        new UnpartitionedMesh::LightWeightCell(CellType::POLYGON, CellType::QUADRILATERAL);
-
-      // vertex ids:   face ids:
-      //                 2
-      //    3---2      x---x
-      //    |   |     3|   |1
-      //    0---1      x---x
-      //                 0
```

Comment:
can we make sure that the comments that describe the vertex and face IDs are still somewhere appropriate in the code?

Reply:
What was in `mesh.cc` was copy pasted into `orthogonal_mesh_generator.cc`, see https://github.com/Open-Sn/opensn/blob/c818552e2e8acd0631bdd37d4e3312e4bac742e5/framework/mesh/mesh_generator/orthogonal_mesh_generator.cc#L240-L245